xbmc/cores/VideoPlayer/VideoRenderers/OverlayRendererGUI.cpp

```diff
@@ -148,6 +155,22 @@ COverlayText::COverlayText(CDVDOverlayText * src)
   m_type = TYPE_GUITEXT;
 
   m_layout = nullptr;
+  
+  // Compute the color to be used for the overlay background (depending on the opacity)
+  UTILS::Color bgcolor = bgcolors[CServiceBroker::GetSettings().GetInt(CSettings::SETTING_SUBTITLES_BGCOLOR)];
```

Comment:
doing this in ctor is not ideal. changing settings when video is paused won't show any effect.